pytest_xmlcomp/plugin.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+
+__version__ = "0.2.0"
+
+import pytest
+
+from py.path import local
+import json
+from lxml import etree
+import pluggy
+from pytest_xmlcomp import hooks
+
+#hookspec = pluggy.HookspecMarker("pytest_xmlcomp")
+#hookimpl = pluggy.HookimplMarker("pytest_xmlcomp")
+
+
+#pm = pluggy.PluginManager("pytest_xmlcomp")
+#pm.add_hookspecs(hooks.TransformSpec)
+#pm.register(hooks.TransformXML())
+
+def pytest_addoption(parser):
+    group = parser.getgroup('xmlcomp')
+    group.addoption(
+            '--datadir',
+            action='store',
+            dest='datadir',
+            default='tests/data',
+            help='Give the directory containing the XML files.')
+
+def pytest_addhooks(pluginmanager):
+    pluginmanager.add_hookspecs(hooks)
+    pluginmanager.register(hooks, "pytest")
```

Comment:
I guess, the prefix (here "pytest") is the prefix of your plugin. At least, it is what I see when I look into the pytest source code. I would suggest to use "xmlcomp".